### PR TITLE
style(Skeleton): remove default children of Skeleton.Node

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/50138

### 💡 Background and solution

Skeleton.Node 内建了一个 `<DotChartOutlined />`，语义比较奇怪，这个图标和 Node 没什么联系。

而且演示中也重复引用了 `<DotChartOutlined />`，猜测是第一版做的时候误解了设计稿。

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Make Skeleton.Node custom node by remove it's default icon children. |
| 🇨🇳 Chinese | 移除 Skeleton.Node 默认的图标 `children` 使其成为真正的自定义节点。|

### CODE REVIEW

**Comment by coding-ice** [comment]
一眨眼，就被你做完了

**Review by li-jia-nan — DISMISSED** [review]

File: components/skeleton/style/index.ts [hunk 1]
```diff
@@ -147,10 +147,10 @@ const genSkeletonElementImage = (token: SkeletonToken): CSSObject => {
   const { skeletonImageCls, imageSizeBase, gradientFromColor, borderRadiusSM, calc } = token;
   return {
     [`${skeletonImageCls}`]: {
```
Comment (li-jia-nan) [inline components/skeleton/style/index.ts:149]:
```suggestion
    [skeletonImageCls]: {
```

Comment (li-jia-nan) [reply components/skeleton/style/index.ts]: 我突然想到这个在 master 改过了，branch merge 一下应该就好了，这里改掉会不会有冲突

Comment (afc163) [reply components/skeleton/style/index.ts]: 改成一样不会有冲突。

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**